src/components/Nav/Nav.js

```diff
@@ -1,8 +1,78 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import MenuList from './Menu/MenuList';
+import {
+  MdSearch,
+  MdOutlineFavoriteBorder,
+  MdOutlineShoppingBag,
+} from 'react-icons/md';
+import './Nav.scss';
 
 export class Nav extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      categories: [],
+    };
+  }
+  componentDidMount() {
+    this.getMockUpCategoriesData();
+  }
+
+  getMockUpCategoriesData() {
+    fetch('http://localhost:3000/data/ysLim/navCategories.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          categories: data,
+        });
+      });
+  }
   render() {
-    return <div>"Nav"</div>;
+    const { toggle } = this.props;
+    const { categories } = this.state;
+    return (
+      <header>
+        <div className="preHeader">
+          <ul>
+            <li>고객센터</li>
+            <li>맴버가입</li>
+            <li onClick={toggle}>로그인</li>
+          </ul>
+        </div>
+        <div className="header">
+          <nav>
+            <MenuList categories={categories} />
+            <div className="logo">
+              <Link
+                to={{
+                  pathname: `/main`,
+                }}
+              >
+                <img src="images/LogoLIKE.svg" alt="라이키 로고" />
```

Comment:
```suggestion
                <img src="/images/LogoLIKE.svg" alt="라이키 로고" />
```
절대경로로 적어주세요! 상대경로로 지정하게 되면 path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!

예시) 상대경로로 작성했을 시
- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.